runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/BatchDataflowWorker.java

```diff
@@ -214,59 +174,14 @@ protected BatchDataflowWorker(
 
     this.memoryMonitor = MemoryMonitor.fromOptions(options);
     this.statusPages =
-        WorkerStatusPages.create(
-            DEFAULT_STATUS_PORT, this.memoryMonitor, sdkHarnessRegistry::sdkHarnessesAreHealthy);
+        WorkerStatusPages.create(DEFAULT_STATUS_PORT, this.memoryMonitor, () -> true);
```

Comment:
what does this "always return true" lambda indicate?

Reply:
The existing implementation would be considered healthy if there were any SDK harnesses. In the case of the legacy-worker (aka Java production runner) it would use an empty SDK harness registry which returned true: https://github.com/apache/beam/blob/4ffeae4d2b800f2df36d2ea2eab549f2204d5691/runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/SdkHarnessRegistries.java#L265

Instead of removing this feature I feel as though in the future this health check could rely on an actual attribute which is why I left it with this trivial lambda.